accounts/usbwallet/wallet.go

```diff
@@ -67,6 +67,10 @@ type driver interface {
 	// SignTx sends the transaction to the USB device and waits for the user to confirm
 	// or deny the transaction.
 	SignTx(path accounts.DerivationPath, tx *types.Transaction, chainID *big.Int) (common.Address, *types.Transaction, error)
+
+	// SignData sends a blob of data to the USB device and waits for the user to confirm
+	// or deny the transaction.
+	SignData(path accounts.DerivationPath, hash []byte) (common.Address, []byte, error)
```

Comment:
I'm unsure about passing a `hash` vs. the data itself. Won't some hardware wallet or Clef itself need the entire data?

Reply:
actually the message itself is passed, I'm going to change the variable name.

Reply:
Please do :)

Reply:
Parameter name is still hash